go.mod

```diff
@@ -10,78 +10,85 @@ require (
 	github.com/openshift/cluster-network-operator v0.0.0-20221129140819-4cbcf0da9cb8
 	github.com/openshift/operator-custom-metrics v0.5.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.55.0
-	github.com/prometheus/client_golang v1.12.2
+	github.com/prometheus/client_golang v1.15.1
 	github.com/stretchr/testify v1.8.1
-	k8s.io/api v0.25.2
-	k8s.io/apimachinery v0.25.2
-	k8s.io/client-go v0.25.2
-	sigs.k8s.io/controller-runtime v0.13.0
+	k8s.io/api v0.27.2
```

Comment:
Can we test this on a 4.10 cluster?

Reply:
I was able to run it locally against a 4.10 cluster. I have not tried redeploying it over the current deployment since I have to stop the syncing.